_posts/2022-08-19-dev-sprint-attendees.md

```diff
@@ -0,0 +1,44 @@
+
+## Adjusting Your Travel/Lodging Details: Act Early!
+If you're planning to stay for the sprints *(this year, October 20-21)*, you'll want to make your travel & lodging plans to stay through Thursday or Friday (depening on how long you want to sprint).
+
+## What should you know if you want to sprint?
+### Git and Github basics
+Most FLOSS projects these days use Git for version control (the software used for keep track on changes to software). This makes it easier for groups of people to work on the same software, together. If you're new to Git, I recommend the first three chapters of [Pro Git](https://git-scm.com/book/en/v2), which is freely available to read online or to download. Those chapters will get you started and give you about 80% of the Git you'll use most often. Chapters five and six, on [Distributed Git](https://git-scm.com/book/en/v2/Distributed-Git-Distributed-Workflows) and [Github](https://git-scm.com/book/en/v2/GitHub-Account-Setup-and-Configuration) respectively, are also very helpful if you're new to working with a group on software.
+
+If you're new to Git or working on software in teams, a basic knowledge of Git fundamentals is important to HELPING maintainers. Keep in mind that one of the main focuses of contributing to a FLOSS project is to help the people who maintain the software we all use. So, while the sprints aren't the best place to *learn* Git, they are a great opportunity to *practice* or improve your Git skills, especially in situations you may not run into normally.
+
+In the spirit of helping our maintainers, try to focus on learning to make "good" commits. More specifically, you want to make [atomic commits](https://www.google.com/search?q=git+atomic+commits&hl=en&sxsrf=ALiCzsZ5lCsXtqzu8xcWZKon8euYTr5VHQ%3A1660677673504&source=hp&ei=Ke77Yq-XF5HXkPIPgbii-Ag&iflsig=AJiK0e8AAAAAYvv8OWWkVH4exyHR-_FtDC8MqJj9RTpd&ved=0ahUKEwjvz7u2isz5AhWRK0QIHQGcCI8Q4dUDCAo&uact=5&oq=git+atomic+commits&gs_lcp=Cgdnd3Mtd2l6EAMyBAgjECcyBggAEB4QFjIGCAAQHhAWMgYIABAeEBYyBggAEB4QFjIFCAAQhgMyBQgAEIYDMgUIABCGAzIFCAAQhgM6BwgjEOoCECc6DQguEMcBENEDEOoCECc6BwguEOoCECc6BAgAEEM6CgguEMcBENEDEEM6EAguELEDEIMBEMcBENEDEEM6CggAELEDEIMBEEM6EQguEIAEELEDEIMBEMcBENEDOgsIABCABBCxAxCDAToQCAAQgAQQhwIQsQMQgwEQFDoICAAQgAQQsQM6CggAEIAEEIcCEBQ6BQgAEIAEUPIIWKIiYMMoaAJwAHgAgAGLAYgB9RCSAQQzLjE2mAEAoAEBsAEK&sclient=gws-wiz) (each commit is focused around a specific activity or code change) and clear commit messages (the changes in a commit should be clear from its commit message). This is important because commits you make will be relyed upon by current or future contributors & maintainers. Making "good" commits and messages is both an art and a science, so practice or experience helps. You should also have a Github account. Git and Github, while related, are two different things, so getting Git setup on your computer won't give you a Github account, just as creating a Github account doens't mean you'll have Git installed on your computer.
```

Comment:
```suggestion
In the spirit of helping our maintainers, try to focus on learning to make "good" commits. More specifically, you want to make [atomic commits](https://www.google.com/search?q=git+atomic+commits), in which each commit is focused around a specific activity or code change, and clear commit messages, meaning that the changes in a commit should be clear from its commit message. This is important because commits you make will be relied upon by current or future contributors & maintainers. Making "good" commits and messages is both an art and a science, so practice or experience helps. You should also have a GitHub account. Git and GitHub, while related, are two different things, so getting Git setup on your computer won't give you a GitHub account, just as creating a GitHub account doesn't mean you'll have Git installed on your computer.
```

Tweaked the link, some verbiage, and a couple spelling fixes.